CONTRIBUTING.md

```diff
@@ -389,6 +390,17 @@ high value in its `priority` (greater than 100 is recommended) to avoid the erro
 
 Also, we can hide an element from the view using `invisible="1"`.
 
+#### Demo Records
+
+Suffix all demo record XML IDs with `demo`. This allows them to be easily
+distinguished from regular records, which otherwise requires examining the
+source or reinstalling the module with demo data disabled.
+
+```xml
+<record id="not_a_real_user_demo" model="res.users">
```

Comment:
There are some situations in which you may only have an external ID - although I can't think of one off hand, it's definitely happened.

Hmmm you're right about there not being anything regarding data records in the guidelines though - both demo or non. Wouldn't hurt to expand the scope of this PR for all data records, instead of just Demo.